### ISSUE TITLE
Dbeaver generates wrong DDL for views with no schema binding

### ISSUE THREAD

**Issue by sravikrishna3** [dbeaver/dbeaver#8701]
dbeaver ver 7.0.4 on mac.
Database: AWS Redshift

Found an easily reproducible issue with dbeaver Generate DDL.

Case 1: A view is created as follows
            create view dba.ravitest as 
            select * from operational_analytics.actual_calls_pod

When we right click the view and generate the DDL it generates it as
         CREATE OR REPLACE VIEW dba.ravitest
         AS SELECT actual_calls_pod.region_id, actual_calls_pod.region_name, 
        actual_calls_pod.district_id, actual_calls_pod.district_name, actual_calls_pod.footprint_id, 
        actual_calls_pod.terr_name, actual_calls_pod.p1_actuals, actual_calls_pod.p1_calls_vhh
        FROM operational_analytics.actual_calls_pod
which is correct and can be replayed later to create the view somewhere else.

Case 2: Same view as before, this time with a clause "with no schema binding".

        create view dba.ravitest as 
       select * from operational_analytics.actual_calls_pod
       with no schema binding

The DDL generated by dbeaver is

    CREATE OR REPLACE VIEW dba.ravitest
     AS create view dba.ravitest as  <--- problem line
   select * from operational_analytics.actual_calls_pod
   with no schema binding

This time it creates a bad ddl by adding create view two times 
This DDL can not be used without manually editing it.

**Comment by uslss** [comment]
thanks for the bug report

**Comment by serge-rider** [comment]
Fixed

**Comment by uslss** [comment]
verified

**Comment by sravikrishna3** [comment]
I see that this is fixed in 7.1.1

**Comment by sravikrishna3** [comment]
thanks

**Comment by pgiansante** [comment]
I still see this issue in Version 21.0.0.202103102007.

**Comment by sravikrishna3** [comment]
I am using ver 21.0.1 and it shows view ddl as

create view cam_laad.cam_cohort_builder_rx
AS SELECT * FROM cam_laad_ext.cam_cohort_builder_rx with no schema binding

Looks right to me.

**Comment by pgiansante** [comment]
Here are my steps:

  1.  If I generate full DDL on a schema, I currently see the issue of 2 “create or replace view as”.  I guess left over from the bug.
  2.  I copied the DDL into a script window and went through cleaning up the multiple “create views”.
  3.  I executed the entire script.
  4.  After refreshing the views, I regenerated the full DDL and I still see the multiple “create views”.

I noticed if I execute one view at a time, it fixes the issue.

It is executing the entire script vs. one at time that is not addressing the issue.

Maybe a different bug?
